drug-drug-interaction/README.md

```diff
@@ -0,0 +1,41 @@
+### Drug-Drug Interaction
+
+## Problem
+In this example, we attempt to infer unknown drug drug interactions from a network of
+multiple drug-based similarities and known interactions.
+
+## Dataset
+There are three datasets which are derived from [DrugBank 4.3](https://go.drugbank.com/downloads/archived) and [drugs.com](Drugs.com).
```

Comment:
Still inconsistent line breaks (arbitrary breaks, clause breaks, and no breaks).
Pick one and stick with it.